scripts/slurm_acc.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
 
+# This script can be run to extract usage accounting data from SLURM for later
+# parsing by an APEL client.
+
 sleep 2
 
 NOW=$(date +"%Y%m%d")
 FILE="/var/log/apel/slurm_acc.$NOW"
 
+# This is the original sacct call which uses CPUTimeRAW for CPU Duration.
```

Comment:
Is the intention here sites would use one `sacct` command or the other? If so, maybe a comment stating that would be helpful.

Reply:
👍